tests/test_client.py

```
@@ -0,0 +1,520 @@
+    {"referenceCode": "ATLAS-CONF-2024-001", "type": "ConfNote"},
+]
+
+SAMPLE_TRIGGERS = [
+    {"name": "HLT_e60_lhmedium", "category": {"name": "electron", "year": "2024"}},
+]
+
+
+@pytest.fixture
+def glance(test_settings: StareSettings) -> Glance:
+    """Glance instance using a fixed token (no real auth)."""
+    return Glance(settings=test_settings, token="fake-token")
```

Ruff (0.15.10)

[error] Possible hardcoded password assigned to argument: "token"

(S106)